### DISCUSSION TITLE
Cypress 10 - How can I get the configured baseUrl?

### DISCUSSION THREAD

**Discussion by andrews1-digital** [cypress-io/cypress#22246] [Questions and Help]
In Cypress 9, I would configure the baseUrl as per [legacy-configuration](https://docs.cypress.io/guides/references/legacy-configuration#cypress-json) and could retrieve its value by `Cypress.config().baseUrl`.
In Cypress 10, I configure the baseUrl in the e2e property of `cypress.config.js`, but now `Cypress.config().baseUrl` returns `null`. Looking at the return of `Cypress.config()`, it seems like I'd need to retrieve its value using `Cypress.config().rawJson.baseUrl`. Is this the recommend approach? Or is there a simpler command to retrieve the configured baseUrl?

**Comment by andrews1-digital** [comment]
Looks like this has been fixed in Cypress 10.1.0 (I was using 10.0.3 when I raised this).

**Comment by rogoit** [comment]
...have same issue with 10.11 in typescript for a validation if it is an internal link

url.includes(Cypress.config('baseUrl')
